plutus-ledger-api/test/Spec/CostModelParams.hs

```diff
@@ -28,7 +28,7 @@ tests =
     [ testCase "length" $ do
             166 @=? length v1_ParamNames
             185 @=? length v2_ParamNames
-            297 @=? length v3_ParamNames
+            303 @=? length v3_ParamNames
```

Comment:
What does this number mean?

Reply:
It means that for each function added (3) there are 2 parameter names introduced to the cost model, making it 6 parameters overall.